site/content/en/reference/live/apply/_index.md

```diff
@@ -269,8 +277,22 @@ DIR:
   This determines the output format of the command. The default value is
   events, which will print the events as they happen. The other option is
   table, which will show the output in a table format.
+
+--server-side:
```

Comment:
Do we want to specify what versions of the api version support this? What's the error message otherwise?

Reply:
@seans3 Can you look at this so we can get this PR merged?

Reply:
Updated with version availability and error message. Please have a look